.circleci/config.yml

```diff
@@ -328,6 +328,31 @@ jobs:
               rm -rf build
               cd conda-build
               python linux_matrix_builder.py --ci_test
+  test_conda_build_osx:
+    macos:
+      xcode: 11.3.0
+    steps:
+      - checkout:
+          path: ./habitat-sim
+      - run:
+          name: Conda Install
+          command: |
+            curl -o ~/miniconda.sh -O https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
+            chmod +x ~/miniconda.sh
+            ~/miniconda.sh -b -p $HOME/miniconda
+            rm ~/miniconda.sh
+            export PATH=$HOME/miniconda/bin:$PATH
+            source ~/.bash_profile
+            conda init "$(basename "${SHELL}")"
```

Comment:
I think this needs to be before `source ~/.bash_profile`.   Alternatively, you can do `source ~/miniconda/etc/profile.d/conda.sh`